src/Tactic/Cong/Id.agda

```diff
@@ -0,0 +1,16 @@
+------------------------------------------------------------------------
+-- The Agda standard library
+--
+-- Version of id used to annotate expressions for use with Tactic.Cong
+------------------------------------------------------------------------
+
+{-# OPTIONS --without-K --safe #-}
+
+module Tactic.Cong.Id where
+
+open import Level
+
+⌊_⌋ : ∀ {ℓ} {A : Set ℓ} → A → A
+⌊ a ⌋ = a
```

Comment:
What's the reason this lives in it's own file? I think we could quite happily put it in `Tactic.Cong`

Reply:
Turns out that it causes problems if it is defined directly inside the `Tactic.Cong` module, since internally Agda turns all definitions inside a parameterised module into definitions with extra parameters. This makes it much more awkward to refer to `⌊_⌋` in reflected terms, since these extra arguments must be supplied, and if you want to do this from within `Tactic.Cong`, supplying these arguments becomes quite circular.

I do wonder whether it would not extra-parameterise `⌊_⌋` if I put it _above_ the `module` line in Tactic.Cong. I'll try that.

Reply:
I now have this definition living in Tactic.Cong.Common, along with some other useful definitions. One benefit of keeping this separate is that you can have multiple instances of Tactic.Cong opened in scope (with renamed versions of `_≈⌊_⌋_`) without having multiple copies of these other definitions around causing ambiguity errors.